web-api/src/Kerbero.Identity/Modules/Users/Services/UserService.cs

```diff
@@ -96,6 +98,22 @@ public async Task<UserReadDto> Create(UserCreateDto createDto)
     return UserMappings.Map(user);
   }
 
+  public async Task ConfirmEmailAsync(Guid userId, string code)
+  {
+    var user = await _userManager.GetById(userId);
+    if (user == null)
+    {
+      throw new UnauthorizedException();
+    }
+    
+    code = Encoding.UTF8.GetString(WebEncoders.Base64UrlDecode(code));
```

Comment:
```suggestion
    code = Encoding.UTF8.GetString(WebEncoders.Base64UrlDecode(code));
```
it is really need to encode/decode the confirm email "code"?

Reply:
The microsoft online example did it. I copied, but not seems to be a bad idea.